components/supervisor-api/status.proto

```diff
@@ -72,6 +72,13 @@ service StatusService {
         };
     }
 
+    // ResourcesStatus provides workspace resources status information.
+    rpc ResourcesStatus(ResourcesStatuRequest) returns (ResourcesStatusResponse) {
```

Comment:
@csweichel Could you have a look at new status endpoint please?

In this PR it respects only cgroup v1. I will open an issue to respect cgroup v2 as a follow-up.

Reply:
I added `top` only to `/.supervisor/supevisor`, not yet to `gp`.

https://github.com/gitpod-io/gitpod/issues/9521 mention though ideally we should have on `gp` too. Would it be alright with you to add it there?  cc @loujaybee

Reply:
for now I will stick with internal cli, there are some uncertainties about implementation, so let's explore first